### PR TITLE
Fix alt:V Multiplayer naming as per their branding

### LINKED ISSUE #600: bug: alt:V incorrect naming
**Describe the bug**
alt:v is listed as AltVMP this is against there branding guidelines as per here https://altv.mp/branding/

**Expected behavior**
Correct use of the games name

**Screenshots or Data**
image

### PR DESCRIPTION
**Pull request by xCausxn**
Fixes #600

### CODE REVIEW

**Comment by xCausxn** [comment]
I can modify the ID to be `gta5am` and add an old_id to be be `gta5a` to satisfy the ID test. thoughts @CosminPerRam

**Comment by CosminPerRam** [comment]
@xCausxn yep, please do as its needed

**Comment by xCausxn** [comment]
Sorted

File: GAMES_LIST.md [hunk 1]
```diff
@@ -423,12 +423,12 @@ Pass the login into to GameDig with the additional options: login, password
 If you are using a FQDN for your server, you will need to set the host parameter to be this domain e.g. rage2.mydomain.com
 This is due to how the Rage MP master server works with server ids as the ip is only used in the ID if no FQDN is provided.
 
-### <a name="gta5a"></a> Grand Theft Auto V - AltV MP
-When querying a server on AltV MP, you have two options for querying the server:
+### <a name="gta5a"></a> Grand Theft Auto V - alt:V Multiplayer
```
Comment (CosminPerRam) [inline GAMES_LIST.md:426]: `name` should be `gta5am` now

**Review by CosminPerRam — CHANGES_REQUESTED** [review]

**Review by CosminPerRam — APPROVED** [review]
Thanks!

**Pull request merged**